README.md

```diff
@@ -1,2 +1,28 @@
 # agent-packaging
 Combined packaging scripts for the Stackdriver monitoring agent
+
+## Branches in Use
+
+* master
+
+  We use this branch to build the 6.X.X agent DEB and RPM packages for the Cloud Rapture repos.
+  
+* stackdriver-agent-5.5.2
+
+  We use this branch to build the 5.X.X agent DEB and RPM packages for the Cloud Rapture repos.
+
+* legacy-agent-rpm
+  
+  We use this branch to build the RPM package for the legacy repo.stackdriver.com repo.
+  
+* legacy-agent-deb
+  
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
```

Comment:
[Optional] `the 5.5.2-X DEB package`...

Reply:
Done

Reply:
Sorry, still not done, unless I'm missing a commit. I feel it's important to mention so we don't accidentally try to build a 6.0.0 package using those build scripts. WDYT?

Reply:
Seems like in the last round I only replied `Done` for places I did not change... This one I'd prefer not to distinguish with finer granularity, as there is no plan to have another branch with `6.0.0` for legacy package. `legacy-agent-deb` will be the only one for legacy DEB.